src/curl.ts

```diff
@@ -0,0 +1,40 @@
+import { BeeRequest, Utils } from '@ethersphere/bee-js'
+import { Readable } from 'stream'
+import { printer } from './printer'
+
+const getDataString = (data?: unknown): string | null => {
+  if (typeof data === 'string') {
+    return data
+  }
+
+  if (data instanceof Readable) {
+    return '<stream>'
+  }
+
+  if (data instanceof Uint8Array) {
+    return '<buffer>'
+  }
+
+  return '<unknown>'
+}
+
+function printCurlCommand(request: BeeRequest): BeeRequest {
```

Comment:
Damn, I am an idiot 🙄. The hook should not be required to return anything, I have just wrote wrong type for it...

Reply:
No biggie. 🙂